components/brave_wallet_ui/components/desktop/views/portfolio/portfolio-asset.tsx

```diff
@@ -771,14 +770,12 @@ export const PortfolioAsset = (props: Props) => {
                 <ArrowIcon
                   isDown={isSelectedAssetPriceDown}
                 />
-                <PercentText>
-                  {
-                    selectedAssetFiatPrice
-                      ? Number(selectedAssetFiatPrice.assetTimeframeChange)
-                        .toFixed(2)
-                      : 0.00
-                  }%
-                </PercentText>
+                {
+                  selectedAssetFiatPrice
+                    ? Number(selectedAssetFiatPrice.assetTimeframeChange)
+                      .toFixed(2)
+                    : 0.00
```

Comment:
Nit: should be a string, since `toFixed()` also returns a string.

Reply:
Changed to string!